sdk/CMakeLists.txt

```diff
@@ -179,13 +140,18 @@ add_custom_command(
     ${_flatcc_source_dir}/bin/flatcc -cwr -o
     ${_program_schema__include_dir}/executorch/sdk/etdump
     ${_etdump_schema__srcs}
-  COMMAND rm -f ${_etdump_schema_cleanup_paths}
-  WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/sdk
```

Comment:
Even when building ahead of time, the flatcc build will still put the host binaries in `_etdump_schema_cleanup_paths`. If we don't delete those here, then we need to be sure that the cross-compiled binaries will always overwrite the host binaries.

Did you find that it was necessary to remove this `rm -f` to make things work? Could we keep it to keep things a bit safer for the cross-compiled binaries?